docs/BR.md

```diff
@@ -382,6 +382,10 @@ The Definitions found in the CA/Browser Forum's Network and Certificate System S
 
 **Legal Entity**: An association, corporation, partnership, proprietorship, trust, government entity or other entity with legal standing in a country's legal system.
 
+**Multi-Perspective Issuance Corroboration**: A process by which the determinations made during domain validation and CAA checking by the Primary Network Perspective are corroborated by other Network Perspectives before Certificate issuance. 
+
+**Network Perspective**: Related to Multi-Perspective Issuance Corroboration. A system (e.g., a cloud-hosted server instance) or collection of network components (e.g., a VPN and corresponding infrastructure) for sending outbound Internet traffic associated with a domain control validation method and/or CAA check. The location of a Network Perspective is determined by the point where unencapsulated outbound Internet traffic is first handed off to the network infrastructure providing Internet connectivity to that perspective.
```

Comment:
Regarding the location of a Network Perspective, I agree with the spirit of it, however it might not be feasible for cloud based and similar implementations on large distributed networks. For instance, Google Cloud has network tiers that utilize cold potato routing, which means traffic uses Google’s production network for as long as possible before being handed off to the internet (see https://cloud.google.com/network-tiers/docs/overview). Even for standard tier traffic, depending on factors such as link capacity, service tier, load, etc. traffic could be encapsulated and sent to a peering port geographically far away from where it originated before being handed off to the internet.

I suspect other Cloud providers perform similar optimizations on their network and therefore the location of a perspective based on this definition could be different from the location where the “MPIC code” is running. 

I suggest adding the word “typically” to account for these situations but I’m open to other ideas.

```suggestion
**Network Perspective**: Related to Multi-Perspective Issuance Corroboration. A system (e.g., a cloud-hosted server instance) or collection of network components (e.g., a VPN and corresponding infrastructure) for sending outbound Internet traffic associated with a domain control validation method and/or CAA check. The location of a Network Perspective is determined by the point where unencapsulated outbound Internet traffic is typically first handed off to the network infrastructure providing Internet connectivity to that perspective.
```

Reply:
Thanks!